addons/tmf_loadouts/loadouts/uk_rm_1982.hpp

```diff
@@ -196,9 +193,7 @@ class aar : r {
 class rat : smg {
 	displayName = "Rifleman (AT)";
 	secondaryWeapon[] = {"CUP_launch_M72A6_Loaded"};
-	backpackItems[] = {
-		LIST_2("CUP_launch_M72A6_Loaded")
-	};
+	backpackItems[] = {"CUP_launch_M72A6_Loaded"};
```

Comment:
Fairly sure it is `CUP_launch_M72A6`, it is switched out with the loaded variant on mission start.